specifyweb/frontend/js_src/lib/components/WbToolkit/GeoLocate.tsx

```diff
@@ -166,7 +176,7 @@ export function getSelectedLocalities(
       };
     }
   | undefined {
-  if (!hot) return undefined;
+  if (hot === undefined) return undefined;
```

Comment:
`hot` is never undefined here according to the function signature
don't do needless checks - it's typescript's job to ensure null-safety